### ISSUE TITLE
As a user, I encounter an error when I submit an application after changing from "Renew" to "Initial"

### ISSUE THREAD

**Issue by mmurthydol** [18F/dol-whd-14c#492]
From Harun: _Also, there is a edge case in which if you try to switch from Renewal to Initial (but not the other way around) you will get an error during submission even though the review page says all the validation is correct and okay to submit; there is extra information that needs to be submitted when Renewal is selected in the Application Info section. If you switch from Renewal to Initial without filling out all the data required for a Renewal application, you will get an error on submission. I think something is happening where if you switch from Renewal to Initial, the fields on the formData (on the front end) are not being cleared out. A work around for testing would be to make sure that testers do not switch back and forth between the Renewal and Initial options in the Application Info section and just stick to one option_

Madhu response: _Looks like API call that changes Application from renewal to initial app needs to call a method that clears all the renewal specific fields._

## Acceptance Criteria
- [ ] No error on submission after changing application from renewal to initial application.

#### Accessibility
N/A

**Comment by phirefly** [comment]
workaround will be to implement 'clearing the application' and recommend that action if a user encounters this issue

**Comment by phirefly** [comment]
@mmurthydol talk to AIS about this

**Comment by phirefly** [comment]
@mmurthydol please provide feedback about the viability of updating the error to tell the user to clear the application when encountering this issue

**Comment by mmurthydol** [comment]
@EStriegel Liz, can this defect be postponed to after milestone 1? If not will work with Prabhakar on a potential solution. Right now placed this in ready to start. Am assuming we need this one. Let's discuss as needed.

**Comment by EStriegel** [comment]
@mmurthydol Yes, this can be pushed to after milestone 1.  We have a work around with the "clear application" option if this problem arises, right?

**Comment by PrabhakarThummalaDOL** [comment]
'Clear application' for a Renew Application works fine.
Can be closed.
